10/umbraco-cms/extending/packages/maintaining-packages.md

```diff
@@ -11,70 +9,42 @@ description: >-
 
 ## Manage feature requests and issues
 
-If you want to encourage feedback, feature requests, and issue reports then you should add a forum to your package. You can manage your forums from [your packages profile page](https://our.umbraco.com/member/profile/packages/):
-
-![Link to manage forums](../../../../11/umbraco-cms/extending/packages/images/forums-link.png)
-
-To add a new forum you will need to specify a name and a description, such as:
-
-![Create a new forum](../../../../11/umbraco-cms/extending/packages/images/forum-create.png)
-
-Any forum you create will appear under the Package Files list on the package's public page:
-
-![Forum list](../../../../11/umbraco-cms/extending/packages/images/forums-display.png)
+If you want to encourage feedback, feature requests, and issue reports then you should make available an issue tracker.
 
-## Find collaborators
-
-If you are the sole maintainer of a package then it's a good idea to find someone to help you. If you have accepted pull requests from people then why not ask them if they would like to collaborate?
-
-If someone requests a feature that you think is a good fit but you don't have the time, then ask that person if they would like to work with you to get it added.
-
-If you'd like to find a collaborator you are welcome to raise a 'Request for Collaborators' via the [Umbraco Package Team tracker](https://github.com/umbraco/Umbraco.Packages).
-
-### Add collaborators on Our
-
-You can assign other Our members to the 'team' for your package. Team members will see the package in their list to maintain, and will be able to edit its details. You can manage your team of collaborators from [your packages profile page](https://our.umbraco.com/member/profile/packages/):
-
-![Link to manage team](../../../../11/umbraco-cms/extending/packages/images/team-link.png)
+This can be [provided as information about your package](https://docs.umbraco.com/umbraco-dxp/marketplace/listing-your-package) and will be linked from your package's page on the Umbraco Marketplace. Specifically you should populate the `IssueTrackerUrl` field.
 
 ## Package no longer required?
 
-If your package should no longer be used (perhaps it is now too old, or it has been superseded by another one that you recommend instead), then you should update your package accordingly via [your packages profile page](https://our.umbraco.com/member/profile/packages/):
-
-At the bottom of the 'Package Details' form, tick to say 'Retired' and specify the reason for the retirement.
-
-![Flag package as retired](../../../../11/umbraco-cms/extending/packages/images/flag-as-retired.png)
-
-Moving 'Next' will save your changes, and the retired status and reason will be displayed prominently on the package's public page:
+After time it could be that your package should no longer be used. Perhaps it is now too old, or it has been superseded by another one that you recommend instead.
```

Comment:
```suggestion
After some time it could be that your package should no longer be used. Perhaps it is now too old, or it has been superseded by another one that you recommend instead.
```